### PR TITLE
Converted tests to use imagetool instead of Dockerfile

### PR DESCRIPTION
**Pull request by marinakog**
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9085

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/AuxiliaryImageUtils.java [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+  }
+
+  /**
+   * Check Configured JMS Resource.
+   *
+   * @param domainNamespace domain namespace
+   * @param adminServerPodName  admin server pod name
+   * @param adminSvcExtHost admin server external host
+   */
+  public static void checkConfiguredJMSresouce(String domainNamespace, String adminServerPodName,
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/AuxiliaryImageUtils.java:255]: This utility method should not be part of AuxiliaryImageUtils

Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/AuxiliaryImageUtils.java]: fixed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/AuxiliaryImageUtils.java [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+  }
+
+  /**
+   * Check Configured JDBC Resource.
+   *
+   * @param domainNamespace domain namespace
+   * @param adminServerPodName  admin server pod name
+   * @param adminSvcExtHost admin server external host
+   */
+  public static void checkConfiguredJDBCresouce(String domainNamespace, String adminServerPodName,
```
[the remark this answers is not in the document]
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/AuxiliaryImageUtils.java]: fixed

**Review by anpanigr — COMMENTED** [review]
How about renaming 
   ItMiiAuxiliaryImage.java -->     ItMiiAuxiliaryImage30.java 
   ItMiiAuxiliaryImage40.java --> ItMiiAuxiliaryImage.java
   ItMiiNewCreateAuxImage40.java --> ItAuxImageWithImageTool.java  ( no New in the class) 

and Delete the class 
 ItMiiNewCreateAuxImage.java  because all this ImageTool commands are covered in  ItMiiNewCreateAuxImage40.java

**Review by vanajamukkara — APPROVED** [review]
LG. Run full suite on Jenkins.

**Review by anpanigr — APPROVED** [review]
lgtm

**Pull request merged**